### DISCUSSION TITLE
Next version based on Bootstrap Italia 2

### DISCUSSION THREAD

**Discussion by astagi** [italia/design-angular-kit#127] [Announcements]
🇮🇹 Sono partite le lavorazioni per l'aggiornamento del kit Angular a Bootstrap Italia 2 (basato su Bootstrap 5). 
Sul branch `main` potete trovare il codice dei lavori in corso e nella [milestone Version 1.0.0](https://github.com/italia/design-angular-kit/milestone/3) un piano preliminare delle attività. La vecchia versione è da considerarsi deprecata.
**Commenti e contributi di tutta la community sono più che benvenuti!** 🎉

🇬🇧 Work has begun to update the Angular Kit to Bootstrap Italia 2 (based on Bootstrap 5). 
Branch `main` contains the work in progress, and a preliminary plan of activities can be found in the [Version 1.0.0 milestone](https://github.com/italia/design-angular-kit/milestone/3). The old version of this library has been marked as deprecated.
**Comments and contributions from the whole community are more than welcome!** 🎉

**Comment by stepobiz** [comment]
Domande per contribuire allo sviluppo:
- Come si vuole gestire il nome dell'Ente e le altre informazioni "statiche"?
